technic/machines/LV/solar_panel.lua

```diff
@@ -48,9 +48,11 @@ minetest.register_node("technic:solar_panel", {
 	tiles = {"technic_solar_panel_top.png",  "technic_solar_panel_bottom.png", "technic_solar_panel_side.png",
 	         "technic_solar_panel_side.png", "technic_solar_panel_side.png",   "technic_solar_panel_side.png"},
 	groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2,
-		technic_machine=1, technic_lv=1},
+		  technic_machine=1, technic_lv=1, axey=2, handy=1},
```

Comment:
Minor indent / style
```suggestion
		technic_machine=1, technic_lv=1, axey=2, handy=1},
```